README.md

```diff
@@ -85,7 +85,7 @@ You can specify output builds in a `package.json` as follows:
 ```
 "main": "dist/foo.js",          // CJS bundle
 "umd:main": "dist/foo.umd.js",  // UMD bundle
-"module": "dist/foo.mjs",      // ES Modules bundle
+"module": "dist/foo.m.js",      // ES Modules bundle
```

Comment:
mjs is allowed, it can be read with webpack loader `javascript/auto` but another alternative good name could be dist/foo.module.js but foo.m.js seems quite weird.